SpatialGDK/Documentation/content/glossary.md

```diff
@@ -1,33 +1,119 @@
+*  Maintains the configuration of a [deployment’s](#deployment) currently-loaded [game world](#game-world). (Note that this is the Unreal game world not the [SpatialOS world](#spatialos-world).)<br/> 
+The GSM lists both the URL of the [Map (or Level - see Unreal documentation)](http://api.unrealengine.com/INT/Shared/Glossary/index.html#l) that the [server-workers](#servers) have loaded and the `AcceptingPlayers` flag. (This flag controls whether or not client-servers can spawn anything in the game world.)
+
+>Related:
+>
+> [Server travel]({{urlRoot}}/content/map-travel.md) 
 
 ### GSM
-Short for "Global State Manager".
+Short for [Global State Manager](#global-state-manager).
 
 ### Schema Generator
-An Unreal Editor toolbar command which takes a set of Unreal classes and generates SpatialOS schema that enables automated communication between Unreal and SpatialOS. See the [SpatialOS website documentation’s .schema introduction](https://docs.improbable.io/reference/latest/shared/schema/introduction) for more information.
+An Unreal Editor toolbar command which takes a set of Unreal classes and generates SpatialOS [schema](#schema) that enables automatic communication between Unreal and SpatialOS. 
+
+>Related:
+>[The toolbar]({{urlRoot}}/content/toolbar)
 
 ### Singleton Actor
 A server-side authoritative Actor that is restricted to one instantiation on SpatialOS. See documentation on [Singleton Actors]({{urlRoot}}/content/singleton-actors.md).
 
-[//]: # (Editorial review status: Full review 2018-07-23)
-[//]: # (Issues to deal with, but not limited to:)
-[//]: # (1. Adding more terms)
+### Spatial Type
+Unreal classes have unique properties when running in SpatialOS. Spatial Type (`SpatialType`) is a SpatialOS-specific [class specifier (Unreal documentation)](https://docs.unrealengine.com/en-US/Programming/UnrealArchitecture/Reference/Classes/Specifiers) which reflects this information. There are different categories of Spatial Type, depending on the Actor’s function in your game.
+See the documentation on [Spatial Type](#{{urlRoot}}/content/spatial-type). 
+
+### SpatialType
+See [Spatial Type](#spatial-type).
+
+## SpatialOS terms
+Below is a subset of SpatialOS terms most relevant to the GDK for Unreal. See the [SpatialOS documentation glossary](https://docs.improbable.io/reference/latest/shared/glossary) for a full list of terms specific to SpatialOS.
+
+Note that this SpatialOS documentation glossary assumes you are developing a SpatialOS game using the SpatialOS SDKs ([The Worker SDK and Platform SDK](#https://docs.improbable.io/reference/latest/shared/get-started/working-with-spatialos)),  so it may reference content relevant to that workflow only. While the SpatialOS SDK underpins the GDK for Unreal, the two workflows are not always the same.)
```

Comment:
```suggestion
Note that this SpatialOS documentation glossary assumes you are developing a SpatialOS game using the [Worker SDK and Platform SDK](#https://docs.improbable.io/reference/latest/shared/get-started/working-with-spatialos),  so it may reference content relevant to that workflow only. While some of the concepts underpin the GDK for Unreal, the two workflows are not always the same.
```